examples/cpp/comm/comm_dll/using_comm_dll.cpp

```diff
@@ -82,7 +82,7 @@ int main()
 
     // Exersize Comm facilities
 
-    bond::comm::SocketAddress loopback("127.0.0.1", 25188);
+    bond::comm::SocketAddress loopback("127.0.0.1", EXAMPLE_PORT_1);
```

Comment:
In the examples, do we want an `#ifndef EXAMPLE_PORT_1 #define EXAMPLE_PORT_1 25188 #endif` to make them easier to copy/paste into other build systems?